Monika After Story/game/script-topics.rpy

```diff
@@ -17740,3 +17740,48 @@ label monika_hedonism:
     m 7etd "It's no wonder most people don't follow that belief...{w=0.3}it's too simple, where morality is complicated."
     m 1eud "So it makes sense why Oscar Wilde portrayed hedonism in a bad light."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_convention",
+            category=['conventions? TODO'],
+            prompt="Convention",
+            random=True,
+        )
+    )
+
+
+default persistent._mas_pm_gone_to_comic_con = None
+default persistent._mas_pm_gone_to_anime_con = None
+
+label monika_convention:
+    # TODO - sprite codes
+    m 1eua "Say [mas_get_player_nickname()],"
+    m "have you ever been to a comic convention or anime convention?{fast}{nw}"
+    menu:
+        m "have you ever been to a comic convention or anime convention?"
+        # TODO: responses
+        "Comic convention":
+            $ persistent._mas_pm_gone_to_comic_con = True
+            $ persistent._mas_pm_gone_to_anime_con = False
+        "Anime convention":
+            $ persistent._mas_pm_gone_to_comic_con = False
+            $ persistent._mas_pm_gone_to_anime_con = True
+        "Both!":
+            $ persistent._mas_pm_gone_to_comic_con = True
+            $ persistent._mas_pm_gone_to_anime_con = True
```

Comment:
```suggestion
            $ persistent._mas_pm_gone_to_anime_con = True
            if persistent._mas_pm_watch_mangime:
                m "Oh! I knew you liked anime already, but you like comics too?"
                m "They're a really interesting medium in literature, maybe I should read some more..."
            else:
                m "Oh! I didn't think you liked anime, but it looks like you're a convention fan anyway!"
                m "Not that it's too surprising, the atmosphere of them seems like it could be enjoyable for anyone."
```